apps/backend/src/mutations/ProposalMutations.ts

```diff
@@ -752,4 +757,34 @@ export default class ProposalMutations {
 
     return submitted;
   }
+
+  @Authorized([Roles.USER_OFFICER])
+  async sendEventEmail(
+    agent: UserWithRole | null,
+    args: SendProposalEventEmailArgs
+  ) {
+    if (args.event !== Event.PROPOSAL_SUBMITTED) {

```

Comment:
As I see the endpoint is called `sendProposalEventEmail` but for now it only accept `Event.PROPOSAL_SUBMITTED`. Is this meant to be extended in the future or it is only used to resend emails for `PROPOSAL_SUBMITTED` event.

Reply:
Wanted to limit it at the moment but let me extend it to include other emails.

Reply:
Working on extending it in this pr.